app/utils/client/lib/roomTypes.js

```diff
@@ -38,23 +38,30 @@ export const roomTypes = new class RocketChatRoomTypes extends RoomTypesCommon {
 			rid: roomId,
 		}).count() > 0;
 	}
-	readOnly(rid, user) {
+	readOnly(roomId, user) {
```

Comment:
why did you change the variable name? we are trying to use the same name `rid` for all points